src/client.cpp

```diff
@@ -122,7 +121,7 @@ CClient::CClient ( const quint16  iPortNumber,
     QObject::connect ( &Channel, &CChannel::ConClientListMesReceived, this, &CClient::OnConClientListMesReceived );
     QObject::connect ( &Channel, &CChannel::ConClientListMesReceived, this, &CClient::ConClientListMesReceived );
 
-    QObject::connect ( &Channel, &CChannel::Disconnected, this, &CClient::Disconnected );
+    QObject::connect ( &Channel, &CChannel::Disconnected, this, &CClient::Stop );
```

Comment:
I'm not happy with `Stop` as a name.  What does it _do_?  It doesn't exit the application...

It's used (at least) here and in the `SIGINT`/`SIGTERM` handler.  The latter makes me wonder if it's also used in the `OnAboutToQuit` handling (which should be invoked by the `SIGINT`/`SIGTERM` handler...).

Reply:
Stop() means it stops the client. 
But this concept is not well defined IMO.